### PR TITLE
Replace jpkrohling with yurishkuro for Jaeger components

### PR DESCRIPTION
**Pull request by jpkrohling**
I need to resign from a few components, as I'm not doing a good job in keeping track of what needs to be done for them. Asking around, @yurishkuro volunteered to take over the Jaeger related ones.

Signed-off-by: Juraci Paixão Kröhling <juraci@kroehling.de>

### CODE REVIEW

**Review by fatsheep9146 — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -102,8 +102,8 @@ extension/encoding/zipkinencodingextension/                             @open-te
 extension/headerssetterextension/                                       @open-telemetry/collector-contrib-approvers @jpkrohling
 extension/healthcheckextension/                                         @open-telemetry/collector-contrib-approvers @jpkrohling
 extension/httpforwarder/                                                @open-telemetry/collector-contrib-approvers @atoulme @rmfitzpatrick
-extension/jaegerremotesampling/                                         @open-telemetry/collector-contrib-approvers @jpkrohling @frzifus
-extension/jaegerremotesampling/internal/jaegerremotesamplingdeprecated/ @open-telemetry/collector-contrib-approvers @jpkrohling @frzifus
+extension/jaegerremotesampling/                                         @open-telemetry/collector-contrib-approvers @yurishkuro @frzifus
+extension/jaegerremotesampling/internal/jaegerremotesamplingdeprecated/ @open-telemetry/collector-contrib-approvers @yurishkuro @frzifus
```
Comment (yurishkuro) [inline .github/CODEOWNERS:106]: why keep this line?

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -220,8 +220,8 @@ receiver/hostmetricsreceiver/                                           @open-te
 receiver/httpcheckreceiver/                                             @open-telemetry/collector-contrib-approvers @codeboten
 receiver/iisreceiver/                                                   @open-telemetry/collector-contrib-approvers @Mrod1598 @djaglowski
 receiver/influxdbreceiver/                                              @open-telemetry/collector-contrib-approvers @jacobmarble
-receiver/jaegerreceiver/                                                @open-telemetry/collector-contrib-approvers @jpkrohling
-receiver/jaegerreceiver/internal/jaegerreceiverdeprecated/              @open-telemetry/collector-contrib-approvers @jpkrohling
+receiver/jaegerreceiver/                                                @open-telemetry/collector-contrib-approvers @yurishkuro
+receiver/jaegerreceiver/internal/jaegerreceiverdeprecated/              @open-telemetry/collector-contrib-approvers @yurishkuro
```
Comment (yurishkuro) [inline .github/CODEOWNERS:224]: why keep this line?

File: extension/jaegerremotesampling/internal/jaegerremotesamplingdeprecated/metadata.yaml [hunk 3]
```diff
@@ -10,4 +10,4 @@ status:
   - redhat
   - sumo
   codeowners:
-    active: [jpkrohling, frzifus]
+    active: [yurishkuro, frzifus]
```
Comment (yurishkuro) [inline extension/jaegerremotesampling/internal/jaegerremotesamplingdeprecated/metadata.yaml:13]: why is internal package tracked separately? it's not a module.

**Review by yurishkuro — APPROVED** [review]

**Review by codeboten — COMMENTED** [review]
@yurishkuro the deprecated modules will be removed in this PR: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/27636

**Review by songy23 — COMMENTED** [review]
> Generated code is out of date, please run "make generate" and commit the changes in this PR.

**Review by frzifus — APPROVED** [review]

**Review by codeboten — APPROVED** [review]

**Review by crobert-1 — APPROVED** [review]

**Comment by jpkrohling** [comment]
> Generated code is out of date, please run "make generate" and commit the changes in this PR.

I've been doing the changes manually, given that this doesn't work for me:

```
running go generate ./... in root
go: downloading github.com/aws/aws-sdk-go v1.48.14
go: downloading go.mongodb.org/mongo-driver v1.13.0
verifying go.mongodb.org/mongo-driver@v1.13.0: checksum mismatch
	downloaded: h1:c+OsvIAc3LCdc9dcfowGjT2bWjvLOccxhdguqHJUvbo=
	go.sum:     h1:67DgFFjYOCMWdtTEmKFpV3ffWlFnh+CYZ8ZS/tXWUfY=

SECURITY ERROR
This download does NOT match an earlier download recorded in go.sum.
The bits may have been replaced on the origin server, or an attacker may
have intercepted the download attempt.

For more information, see 'go help module-auth'.
make[1]: *** [Makefile:130: for-all] Error 1
make[1]: Leaving directory '/home/jpkroehling/Projects/src/github.com/open-telemetry/opentelemetry-collector-contrib'
make: *** [Makefile:227: generate] Error 2
```

It might take a couple of more iterations to get this done, sorry about that.

**Review by songy23 — APPROVED** [review]

**Review by TylerHelmuth — APPROVED** [review]

**Pull request merged**